.env.sample

```diff
@@ -1,3 +1,8 @@
 SP_AUTHORIZE_NET_LOGIN_ID: 12345
 SP_AUTHORIZE_NET_TRANSACTION_KEY: IHaveTheSamePasswordOnMyLuggage
 SP_TERRAIN_BASE_URL: https://de.cyverse.org/terrain
+SP_KEYCLOAK_ISSUER: https://kc.cyverse.org/auth/realms/CyVerseTest
+SP_KEYCLOAK_CLIENT_ID: local-testing
+SP_KEYCLOAK_CLIENT_SECRET: NotReal
+NEXTAUTH_URL: https://de.cyverse.org # Only required in production.
+NEXTAUTH_SECRET: NotReal # Add with `npx auth secret`. Read more: https://cli.authjs.dev
```

Comment:
The [NEXTAUTH_SECRET](https://next-auth.js.org/configuration/options#nextauth_secret) env var is used to encrypt the NextAuth.js JWT, and to hash email verification tokens.

I can't remember if this is still required if we're using a [psql database adapter](https://authjs.dev/getting-started/adapters/pg), but I'm also not sure if we need a database adapter to store user sessions for this project 🤔

Reply:
Ah, that's a good point. I'm not sure if we'll need it either. 🤔 Hopefully there's enough room on the secure server for a DBMS or maybe some other reliable session store.

Reply:
According to their [Database Adapters docs](https://authjs.dev/getting-started/database):

> Auth.js integrations save sessions in a cookie by default. Therefore, setting up a database is optional. However, if you want to persist user information in your own database, or you want to implement certain flows, you will need to use a Database Adapter.

So I guess we can always set up database sessions later if we think we need it.